resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+restartfile=/opt/puppetlabs/server/data/<%= EZBake::Config[:real_name] %>/restartcounter.txt
+cur=$(head -n 1 $restartfile)
+initial=$cur
+timeout="300"
+
+if [ -e $restartfile ];  then
+    /usr/bin/java $JAVA_ARGS -Djava.security.egd=/dev/urandom -cp ${INSTALL_DIR}/<%= EZBake::Config[:uberjar_name] %> clojure.main -m <%= EZBake::Config[:main_namespace] %> --config "${CONFIG}" -b "${BOOTSTRAP_CONFIG}" &
```

Comment:
`/usr/bin/java` needs to be replaced by a variable of some sort. there should be one defined in the default file and used in other commands.